src/base/http/connection.cpp

```diff
@@ -90,6 +90,18 @@ void Connection::read()
             }
             return;
 
+        case RequestParser::ParseStatus::NotImplemented: {
+                Logger::instance()->addMessage(tr("Unsupported Http request, closing socket. IP: %1")
```

Comment:
This should be more specific: "Unsupported Http request method, ..."

Reply:
As in (Also removed closing socket portion as per suggestion in comment below):
``                Logger::instance()->addMessage(tr("Unsupported Http request method. IP: %1")
                    .arg(m_socket->peerAddress().toString()), Log::WARNING);``
Or do you want more information?

Reply:
Hmm, maybe the log message isn't that important, we could omit it. Otherwise https://github.com/qbittorrent/qBittorrent/pull/13072#discussion_r446601113 is also acceptable.

Reply:
Once again... why we really need to pollute log with all this messages about "bad client"?

Reply:
> Once again... why we really need to pollute log with all this messages about "bad client"?

Don't mean to turn this into a discussion about logging, but detailed logging is important for troubleshooting. We might receive bug reports from users that can be solved just by looking at the log, seeing this message, and telling them "Not an issue, your client is misbehaving" (or the user can even figure this out by themselves, before posting an issue report).

Of course, we can't log everything all the time. It would be best to have several logging levels and event types configurable at run-time (i.e. in the settings). This way, we could ask users to turn on detailed logging for all event types before posting an issue report, for example. But this requires a redesign of the logger, which is a matter  for another time.

Right now, personally I don't mind if this particular message isn't logged. Though if this is not logged, then the message in the `case` above it probably shouldn't either, for consistency.

Reply:
I think for the time being I'd leave it in here. We sort of have a configurable log level, but it's more so a filter on the existing log versus actually logging more/less messages.

Reply:
>We might receive bug reports from users that can be solved just by looking at the log, seeing this message, and telling them "Not an issue, your client is misbehaving" 

Can't imagine any client (not abstract HTTP client but the one that uses this particular API) that can misbehave this way. But if you guys insist that it can really help...

Reply:
Also can't imagine how this additional status can help our Web API users.

Reply:
Removed the relevant log messages